addons/mail/static/src/messaging/component/emojis_button/emojis_button.js

```diff
@@ -64,6 +64,7 @@ class EmojisButton extends components.PopoverButtonWithComponent {
             unicode: ev.detail.unicode,
         });
     }
+
```

Comment:
prep, but shouldn't it be deleted? Probably same issue for all deleted files due to copy/paste rebase.

Reply:
looks used by composer though, so I don't think we should remove